app/Http/Controllers/Auth/NewPasswordController.php

```diff
@@ -3,36 +3,47 @@
 namespace App\Http\Controllers\Auth;
 
 use App\Http\Controllers\Controller;
-use App\Http\Requests\Auth\NewPasswordRequest;
+use App\Http\Requests\Auth\ResetPasswordRequest;
 use App\Models\User;
 use Illuminate\Auth\Events\PasswordReset;
+use Illuminate\Http\RedirectResponse;
+use Illuminate\Http\Request;
 use Illuminate\Support\Facades\Hash;
 use Illuminate\Support\Facades\Password;
 use Illuminate\Support\Str;
+use Inertia\Inertia;
+use Inertia\Response;
 
 class NewPasswordController extends Controller
 {
-    /**
-     * Reset Password.
-     *
-     * @throws \Illuminate\Validation\ValidationException
-     */
-    public function store(NewPasswordRequest $request)
+    public function create(Request $request): Response
     {
+        return Inertia::render('Auth/ResetPassword', [
+            'email' => $request->email,
+            'token' => $request->route('token'),
+        ]);
+    }
+
+    public function store(ResetPasswordRequest $request): RedirectResponse
+    {
+        $request->validated();
+
         $status = Password::reset(
             $request->only('email', 'password', 'password_confirmation', 'token'),
             function (User $user, string $password) {
-                $user->fill([
+                $user->forceFill([
                     'password' => Hash::make($password),
-                ])->setRememberToken(Str::random(60))
-                ->save();
+                ])->setRememberToken(Str::random(60));
+
+                $user->save();
 
                 event(new PasswordReset($user));
             }
         );
 
         return $status === Password::PASSWORD_RESET
-            ? response()->json(['status', __($status)])
-            : response()->json(['error' => [__($status)]]);
```

Comment:
You are overriding the controller which is used for the admin panel API. Please create a separate controller and move over your code there. Make sure to structure it so that the admin panel API and the main platform are separated nicely.